roles/iridl/templates/docker-compose.yaml

```diff
@@ -39,6 +39,15 @@ services:
     volumes:
       - "{{maproom_install_dir}}:/var/www/html:rslave,ro"
 
+{% if python_maproom_repo is defined %}
+  python_maproom:
+    image: python_maproom:{{python_maproom_version}}
+    expose:
+      - 8000
+    restart: always
+    volumes:
```

Comment:
currently thinking about what the best way to handle volumes is

Reply:
Use environment variable instead of config file for data location.